apps/heft/src/metrics/MetricsCollector.ts

```diff
@@ -151,8 +153,8 @@ export class MetricsCollector {
       totalUptimeMs: process.uptime() * 1000,
       machineOs: process.platform,
       machineArch: process.arch,
-      machineCores: os.cpus().length,
-      machineProcessor: os.cpus()[0].model,
+      machineCores: cpus.length,
+      machineProcessor: cpus[0].model,
```

Comment:
Trim?

Reply:
Not sure what you're asking. This code is deliberately collecting the processor model for analytics. We could turn that off and/or only collect it once, but that would be a behavior change whereas the rest of this PR is not.